hw4.Rmd

```diff
@@ -30,43 +37,103 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+# languages less likely to have an appeal granted?
+dat %>%
+  select(language, decision) %>%
+  group_by(language) %>%
+  summarise(yes = mean(decision=="yes")) %>%
```

Comment:
Using the default stat for geom_bar() instead of "identity" makes this unnecessary. Also, I'm clear why you are grouping by language.